op-guide/offline-ansible-deployment.md

```diff
@@ -260,13 +243,7 @@ Descriptions about the two circumstances are as follows.
         # ansible_user = tidb
         ```
     
-    2. Connect to the network and download TiDB binary to the Control Machine.
-
-        ```
-        ansible-playbook local_prepare.yml
-        ```
-    
-    3. Initialize the system environment and edit the kernel parameters.
+    2. Initialize the system environment and edit the kernel parameters.
```

Comment:
Item number 4？

Reply:
Just leave it as "2". Please pay attention to the line number on the left.